larpix/format/hdf5format.py

```diff
@@ -57,6 +57,118 @@
 The file data is saved in HDF5 datasets, and the specific data format
 depends on the LArPix+HDF5 version.
 
+Version 2.0 description
+^^^^^^^^^^^^^^^^^^^^^^^
+
+For version 2.0, there are two dataset: ``packets`` and ``messages``.
+
+The ``packets`` dataset
+contains a list of all of the packets sent and received during a
+particular time interval.
+
+    - Shape: ``(N,)``, ``N >= 0``
+
+    - Datatype: a compound datatype (called "structured type" in
+      h5py/numpy). Not all fields are relevant for each packet. Unused
+      fields are set to a default value of 0 or the empty string.
+      Keys/fields:
+
+        - ``io_group`` (``u1``/unsigned byte): an id associated with the
+          high-level io group associated with this packet
+
+        - ``io_channel`` (``u1``/unsigned byte): the id associated with the
+          mid-level io channel associated with this packet
+
+        - ``packet_type`` (``u1``/unsigned byte): the packet type code, which
+          can be interpreted according to the map stored in the
+          'packets' attribute 'packet_types'
+
+        - ``chip_id`` (``u1``/unsigned byte): the LArPix chip id
+
+        - ``parity`` (``u1``/unsigned byte): the packet parity bit (0 or
+          1)
+
+        - ``valid_parity`` (``u1``/unsigned byte): 1 if the packet
+          parity is valid (odd), 0 if it is invalid
+
+        - ``downstream_marker`` (``u1``/unsigned byte): a marker to indicate the
+          hydra io network direction for this packet
```

Comment:
Suggested name: ``hydra_direction``

Reply:
Unless ``downstream_marker`` is the name used in the datasheet

Reply:
downstream marker is what is used in the datasheet, so I think we should stick with that.